github/Organization.py

```diff
@@ -357,6 +357,47 @@ def create_fork(self, repo):
             self._requester, headers, data, completed=True
         )
 
+    def create_repo_from_template(
+            self,
+            name,
+            repo,
+            description=github.GithubObject.NotSet,
+            private=github.GithubObject.NotSet,
+    ):
+        """self.name
+        :calls: `POST  /repos/:template_owner/:template_repo/generate <https://developer.github.com/v3/repos/#create-repository-using-a-repository-template>`_
+        :param name: string
+        :param repo :class:`github.Repository.Repository`
+        :param description: string
+        :param private: bool
+        :rtype: :class:`github.Repository.Repository`
+        """
+        assert isinstance(name, str), name
+        assert isinstance(repo, github.Repository.Repository), repo
+        assert description is github.GithubObject.NotSet or isinstance(
+            description, str
+        ), description
+        assert private is github.GithubObject.NotSet or isinstance(
+            private, bool
+        ), private
+        post_parameters = {
+            "name": name,
+            "owner": self.login,
```

Comment:
I wonder if we should use self._identity here.

Reply:
neither `Organization` nor `AuthenticatedUser` have an `_identity` property